mw-webapp/src/component/inProgress/InProgress.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from "src/component/inProgress/InProgress.module.scss";
+
+const InProgress = () => {
+  return (
+    <div className={styles.container} />
+  );
+};
+
+export default InProgress;
```

Comment:
no default exports, please adjust this rule in eslint for everybody

Reply:
We don't use default export/ Maybe some changes was not pushed?